main/inc/lib/message.lib.php

```diff
@@ -1466,19 +1486,31 @@ public static function showMessageBox($messageId, $type)
 
         switch ($type) {
             case self::MESSAGE_TYPE_OUTBOX:
-                $message_content .= '<a href="outbox.php?'.$social_link.'">'.
-                    Display::return_icon('back.png', get_lang('ReturnToOutbox')).'</a> &nbsp';
-                $message_content .= '<a href="outbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+                    $message_content .= '<a href="outbox.php?'.$social_link.'">'.
+                        Display::return_icon('back.png', get_lang('ReturnToOutbox')).'</a> &nbsp';
+                    $message_content .= '<a href="outbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
+                        Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
+                    break;
             case self::MESSAGE_TYPE_INBOX:
-                $message_content .= '<a href="inbox.php?'.$social_link.'">'.
-                    Display::return_icon('back.png', get_lang('ReturnToInbox')).'</a> &nbsp';
-                $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
-                    Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
-                $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+
+                    $message_content .= '<a href="inbox.php?'.$social_link.'">'.
+                        Display::return_icon('icons/22/arrow_up_1.png', get_lang('ReturnToInbox')).'</a> &nbsp';
+
+                    $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
+                        Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
+                    $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
+                        Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
+                    if ($idPrevMessage != 0) {
+                        $message_content .= '<a title="'.get_lang('ScormPrevious').'" href="view_message.php?type='.$type.'&id='.$idPrevMessage.'" ">'.
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 168 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)